src/UserSpaceInstrumentation/Trampoline.cpp

```diff
@@ -675,6 +780,31 @@ ErrorMessageOr<uint64_t> CreateTrampoline(pid_t pid, uint64_t function_address,
   return address_after_prologue;
 }
 
+uint64_t GetReturnTrampolineSize() {
+  // The size is constant. So the calculation can be cached on first call.
+  static const uint64_t return_trampoline_size = []() -> uint64_t {
+    MachineCode unused_code;
+    AppendCallToExitPayloadAndJumpToReturnAddress(0 /*exit_payload_function_address*/, unused_code);
```

Comment:
I think that in general this should be
```suggestion
    AppendCallToExitPayloadAndJumpToReturnAddress(/*exit_payload_function_address=*/0, unused_code);
```

so that clang-tidy recognizes it as the parameter name. I think.